syntax = "proto3";

package avant;

option go_package = "/proto_res";

import "proto_cmd.proto";

// ProtoCmd
message ProtoPackage
{
    ProtoCmd cmd = 1;
    bytes protocol = 2;
}
